### PR TITLE
SASS-4100 - Update README.md file for State Benefits BE

### PR DESCRIPTION
**Pull request by satyaayinala**
### Description
Update README.md file for State Benefits BE

Add a link to the relevant story in Jira
[SASS-4100](https://jira.tools.tax.service.gov.uk/browse/SASS-4100)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you run the smoke tests?
- [ ]  Have you run the performance tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you run the smoke tests? (where applicable)
- [ ]  Have you run the performance tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

**Comment by georgievh** [comment]
Can you please update the SBT version? You don't need to to that for the FE.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,47 @@
...
+The service manager profile for this service is:
+
+    sm --start INCOME_TAX_STATE_BENEFITS
+
+Run the following command to start the remaining services locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+This service runs on port: `localhost:9377`
```
Comment (georgievh) [inline README.md:21]: Can you pleaes move this just below:
sm --start INCOME_TAX_STATE_BENEFITS

File: README.md [hunk 2]
```diff
@@ -3,6 +3,47 @@
...
+
+### Downstream services
+All State Benefits data is retrieved / updated via the downstream system.
+- IF (Integration Framework)
```
Comment (georgievh) [inline README.md:25]: You should include income-tax-submission too

File: README.md [hunk 3]
```diff
@@ -3,6 +3,47 @@
...
+
+### State Benefits Sources (HMRC-Held and Customer Data)
+State Benefits data can come from different sources: HMRC-Held and Customer. HMRC-Held data is state benefits data that HMRC have for the user within the tax year, prior to any updates made by the user. The state benefits data displayed in-year is HMRC-Held.
+
+Customer data is provided by the user. At the end of the tax year, users can view any existing state benefits data and make changes (create, update and delete).
+
+Examples can be found here in the [income-tax-submission-stub](https://github.com/hmrc/income-tax-submission-stub/blob/main/app/models/StateBenefitsUsers.scala)
```
Comment (georgievh) [inline README.md:32]: Probably you should say something like:
...
Examples of prior user data can be found ...
...

**Review by georgievh — CHANGES_REQUESTED** [review]

**Comment by georgievh** [comment]
Please include a Feature Switches section and add Encryption

File: README.md (hunk 2 above)
Comment (satyaayinala) [reply README.md:25]: Added

File: README.md (hunk 3 above)
Comment (satyaayinala) [reply README.md:32]: Changed text

**Review by georgievh — APPROVED** [review]

**Pull request merged**